components/operator/README.md

```diff
@@ -0,0 +1,33 @@
+# Function Operator
+
+The Function Operator is a Kubernetes controller that install Serverless.
+
+## Prerequisites
+The Function Operator requires the following components to be installed:
+- [Go](https://go.dev/)
+
+## Development
+
+### Installing locally build serverless operator image on k3d with serverless
+
+To install serverless-manager from local sources on k3d cluster run:
+```bash
+make install-operator-k3d
```

Comment:
When I do it first time I see error:
`Error from server (NotFound): deployments.apps "serverless-operator" not found`